src/header/mod.rs

```diff
@@ -164,8 +164,9 @@ impl Headers {
     /// # let mut headers = Headers::new();
     /// headers.set_raw("content-length", vec![b"5".to_vec()]);
     /// ```
-    pub fn set_raw<K: Into<Cow<'static, str>>>(&mut self, name: K, value: Vec<Vec<u8>>) {
-        self.data.insert(UniCase(CowStr(name.into())), Item::new_raw(value));
+    pub fn set_raw<'a, K: Into<Cow<'a, str>>>(&mut self, name: K, value: Vec<Vec<u8>>) {
```

Comment:
This change means the example on L165 allocates the `"content-length"` as a `String`, but there's no need.